index.html

```diff
@@ -0,0 +1,53 @@
+<!doctype html>
+
+<html lang= "en">
+  <head>
+    <meta charset= "utf-8">
+    <meta name= "viewport" content= "width=device-width, initial-scale=1">
+    <title> Static Comp Challenge </title>
+    <link rel= "normalize-stylesheet" href= "./node_modules/normalize.css/normalize.css">
```

Comment:
Not sure how to use/link these style sheets. Play with this more as styling gets underway